Testing/SystemTests/tests/analysis/VesuvioCommandsTest.py

```diff
@@ -58,9 +58,9 @@ def _create_test_flags(background, multivariate=False):
     else:
         mass1 = {'value': 1.0079, 'function': 'GramCharlier', 'width': [2, 5, 7],
                  'hermite_coeffs': [1, 0, 0], 'k_free': 0, 'sears_flag': 1}
-    mass2 = {'value': 16.0, 'function': 'Gaussian', 'width': 10}
```

Comment:
This is going to break the Vesuvio user scripts and I don't think the scientists will like having to number the functions themselves. Is this necessary?

Reply:
not sure how best to do that and I am open to suggestions.

Reply:
I think Users will still be able to carry out their immediate work needs if they just use the previous mantid version, while in the meantime we help them update their scripts. It's a simple change, anyway.

Reply:
If we leave the script here specifying `Gaussian` then I think we just need to update the reference results to contain the new text axis values?